articlequality/feature_lists/ptwiki.py

```diff
@@ -0,0 +1,169 @@
+"""
+Portuguese Wikipedia
+++++++++++++++++++++
+"""
+import re
+
+from revscoring import Feature
+from revscoring.datasources.meta import filters, mappers
+from revscoring.features import wikitext
+from revscoring.features.meta import aggregators
+from revscoring.features.modifiers import log, max, sub
+from revscoring.languages import portuguese
+
+from . import wikipedia
+
+# Templates
+infobox_templates = wikitext.revision.template_names_matching(
+    r"Info", name="ptwiki.revision.infobox_templates")
+CN_TEMPLATES = [
+    r"Carece de fontes",
+    r"Carece de fontes2",
+    r"Carece de fontes/bloco",
+    r"Carece de fontes/bloco2"
+]
```

Comment:
Should we avoid matching "Carece de fontes/bloco/doc", "Carece de fontes falso", and other strings which have one of the items above as prefixes?

Reply:
Interesting.  Do people actually transclude the doc pages?

Reply:
I don't think docs are transcluded outside the Template namespace. I was just considering that was a possibility, and since in this case it is easy to avoid, maybe it is worth doing so.

Reply:
I created https://phabricator.wikimedia.org/T250522 as a follow-up task.